src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -1276,7 +1276,7 @@ public static class Error
       public static readonly string UnsupportedFileType = "The type of file that you are trying to open is not currently supported";
       public static readonly string CannotRemoveBaseGridColumnStillInUse = "Cannot remove base grid column still used by other columns";
 
-      public static string ErrorWhenPlottingDataRepository(int sheetName, string exceptionMessage) => $"Plotting data set number:{sheetName} produced the following error: {exceptionMessage}";
+      public static string ErrorWhenPlottingDataRepository(int sheetName, string exceptionMessage) => $"It was not possible to plot the data sets. Please, update your mappings to correct it. An error occur while plotting data set number:{sheetName + 1} produced the following error: {exceptionMessage}";
```

Comment:
was is it? in update your mapping to correct it?

Reply:
The reason for this error is normally that you are missing a grouping by mapping. So, in 95% of the cases the error is that there is a missing mapping (update your mappings to correct the problem). In the other 5% well, there is a real problem in the data and the time is just not monotonical I would say but I still think it makes sense to draw the attention of the user to the mappings just to check if there are still missing mappings.

Reply:
>Please, update your mappings to correct it. 

I don't think it's english. It needs to be rephrase. What does "it" mean